symplyphysics/laws/electricity/circuits/characteristic_resistance_of_rectangular_waveguide_for_e_mn_waves.py

```diff
@@ -0,0 +1,41 @@
+from sympy import Eq, solve, sqrt
```

Comment:
Decipher `e mn` in file name. Use descriptive names.

Reply:
> Decipher `e mn` in file name. Use descriptive names.

It will be difficult to decipher, because it will not be possible to do it briefly. The designation "e_mn" is generally accepted and means that only the electric field has a longitudinal component and the magnetic field does not. Also, the indices "m" and "n" describe the number of half-integer wavelengths that fit vertically and horizontally across the waveguide cross section, respectively. Different sets of "m" and "n" correspond to different modes of wave propagation in the waveguide. Can I just add the above to the description?

Reply:
Emn are better known as TM, transverse magnetic waves. I suggest to rename e_mn to transverse magnetic and h_mn to transverse electric. Also add small note to description what are TE and TM waves.